src/app/pages/my-observations/components/sent-list/sent-list.component.ts

```diff
@@ -65,98 +64,59 @@ export class SentListComponent implements OnDestroy {
 
     this.myObservationsUrl$ = this.createMyObservationsUrl$();
 
-    addUpdateDeleteRegistrationService.changedRegistrations$
-      .pipe(takeUntil(this.ngDestroy$))
-      .subscribe((newRegistration) => {
-        const regsWithoutNewRegistration = this.loadedRegistrations.filter(
-          (reg) => reg.RegId !== newRegistration.RegId
-        );
-
-        this.loadedRegistrations = [newRegistration, ...regsWithoutNewRegistration];
-
-        this.isEmpty.next(false);
-        this.changeDetectorRef.detectChanges();
+    this.userSettingService.language$.subscribe((langKey) => {
+      const searchCriteria = new BehaviorSubject<SearchCriteriaRequestDto>({
+        OrderBy: 'DtChangeTime',
+        LangKey: langKey,
+      });
+      this.searchResult = this.searchRegistrationService.searchMyRegistrations(searchCriteria);
+    });
+
+    this.searchResult.registrations$
+      .pipe(tap(() => this.scroll && this.scroll.complete()))
+      .subscribe((myRegistrations) => {
```

Comment:
Du henter "mine observasjoner" to ganger fra API'et når du går inn på sida:
![image](https://user-images.githubusercontent.com/71138449/221153089-df6ec761-db2b-4654-bae9-fc712c294b3e.png)